src/VisualStudio/Core/Def/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "EnvDTE": "8.0.0",
+    "EnvDTE": "8.0.1",
```

Comment:
Note that I updated version dependencies for `EnvDTE` and `VSLangProj` in order to work with `VSLangProj2` which was just published. I used Jared's tool to update the entire repo.